core/pkg/opencost/allocation.go

```diff
@@ -174,8 +229,9 @@ func (lba *LbAllocation) SanitizeNaN() {
 // then this type would be unnecessary and its fields would go into the regular Allocation
 // and not in the AggregatedAllocation.
 type RawAllocationOnlyData struct {
-	CPUCoreUsageMax  float64 `json:"cpuCoreUsageMax"`
-	RAMBytesUsageMax float64 `json:"ramByteUsageMax"`
+	CPUCoreUsageMax  float64  `json:"cpuCoreUsageMax"`
+	RAMBytesUsageMax float64  `json:"ramByteUsageMax"`
+	GPUUsageMax      *float64 `json:"gpuUsageMax"` //@bingen:field[version=23]
```

Comment:
I think this is fine, but could create some annoying patterns to double-nil check for a valid `GPUUsageMax` -- Just noting that if we find ourselves writing:

```go
if ra != nil && ra.GPUUsageMax != nil {
    //  ...
}
```
we may want to consider a helper func. Don't see any evidence of that yet, so not sure it's necessary now.